ext/zend_test/test.stub.php

```diff
@@ -17,6 +17,7 @@ class _ZendTestClass implements _ZendTestInterface {
         public int $intProp = 123;
         public ?stdClass $classProp = null;
         public stdClass|Iterator|null $classUnionProp = null;
+        public Traversable&Countable $classIntersectionProp;
```

Comment:
Do you think it's worthwhile to also add a function with intersection param and return types?

Reply:
I'm not sure how to do the check with ZPP or is it possible to them manually?

Reply:
Yeah, I think you should rely on manual param parsing